index.html

```diff
@@ -7,7 +7,7 @@
   <meta charset="UTF-8">
   <title>Emily Bennett Portfolio</title>
   <link rel="stylesheet" href="./style.css">
-
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@900&display=swap" rel="stylesheet">
```

Comment:
Do we need this import statement? The CSS file has two import statements for the two fonts used.

Reply:
I did not add those, I don't know.

Reply:
Right, I had the two fonts imported in the CSS file. Should I change them to the HTML file?

Reply:
I can remove it, if you prefer.